Composer/packages/lib/shared/src/featureFlagUtils/index.ts

```diff
@@ -14,4 +14,13 @@ export const getDefaultFeatureFlags = (): FeatureFlagMap => ({
     isHidden: false,
     enabled: false,
   },
+  ADVANCED_TEMPLATE_OPTIONS: {
+    displayName: formatMessage('Advanced Template Options'),
```

Comment:
nit: The title is not reflective if the flag

not nit: "Advanced template options"

Reply:
I have made the caps change but have not changed the string value, in R14 this will be a hidden feature flag so the text will never be in the UI for this release, if/when it is a visible feature flag then we will take another pass on the copy.